eea/reports/browser/stylesheets/reports.css

```diff
@@ -169,74 +169,35 @@ dl.publicationMessage dd {
     max-width: 100%;
 }
 
-#content .pdfjs #sidebarContainer {
-    width: 20%;
-}
-
-#content .pdfjs #toolbarSidebar {
-    width: 100%;
-}
-
-#content .pdfjs #sidebarContent {
-    width: 100%;
-}
-
-#content .pdfjs #outlineView {
-    padding-right: 0;
-    padding-left: 4px;
-}
-
-#content .pdfjs #outlineView,
-#content .pdfjs #attachmentsView {
-    width: 96%;
-}
-
-html[dir='ltr'] #content .pdfjs .outlineWithDeepNesting>.outlineItem,
-html[dir='ltr'] #content .pdfjs .outlineItem>.outlineItems {
+#content .report-pdf {
     margin-left: 0;
 }
 
-html[dir='ltr'] #content .pdfjs .outlineItem>a {
-    padding-left: 0;
-}
-
-#content .pdfjs .outlineItem>a,
-#content .pdfjs .attachmentsItem>button {
-    font-size: 11px;
-}
-
-#content .pdfjs #thumbnailView {
-    padding: 10px 0 0;
-}
-
-html[dir='ltr'] #content .pdfjs #outerContainer.sidebarOpen>#mainContainer {
-    left: 20%;
-    width: 80% !important;
-}
-
-html[dir='ltr'] #content .pdfjs #outerContainer>#mainContainer {
-    width: 100% !important;
+html[dir='ltr'] .portaltype-report #content .pdfjs .findbar {
+    left: 7%;
 }
 
-html #content .pdfjs #outerContainer .dropdownToolbarButton {
-    width: 70%;
+.portaltype-report #region-content .pdf-viewer-disabled .publicationDescription,
```

Comment:
report.css is loading with a condition that portal_type is Report therefore I want you to simplify all of the selectors you added to maximum 3 items, preferably 1 to 2.
ex:
.pdf-viewer-disabled .publicationDescription 
instead of:
.portaltype-report #region-content .pdf-viewer-disabled .publicationDescription